ETL: Add cloud-to-cloud instructions to InfluxDB I/O tutorial (#88)
crate/cratedb-guide: docs/integrate/etl/influxdb.md

crash --host=cratedb --command="SELECT * FROM testdrive.demo;"
:::

## Cloud to Cloud

> If you're interested in migrating straight from [InfluxDB Cloud] to
> [CrateDB Cloud], the procedure is similar with small adjustments.

Reviewer: On this page, being a technical guideline, we've adjusted our thinking to abstain from the "migration" jargon, and, attaching to the ETL acronym, talk about "data loading", or "data import/export" instead.

```suggestion
If you're interested in importing data from [InfluxDB Cloud] into
[CrateDB Cloud], the procedure is similar with small adjustments.
```

First, helpful aliases again:
:::{code} shell
alias ctk="docker run --rm -it ghcr.io/crate-workbench/cratedb-toolkit:latest ctk"
alias crash="docker run --rm -it ghcr.io/crate-workbench/cratedb-toolkit:latest crash"
:::

You will need your credentials for both CrateDB and InfluxDB. 
These are, with examples:

**CrateDB Cloud**
* Host: ```purple-shaak-ti.eks1.eu-west-1.aws.cratedb.net```
* Username: ```admin```
* Password: ```dZ..qB```

**InfluxDB Cloud**
  * Host: ```eu-central-1-1.aws.cloud2.influxdata.com```
  * Organization ID: ```9fafc869a91a3406```
  * All-Access API token: ```T2..==```

> For CrateDB, the credentials can be downloaded at time of cluster creation,
> and for InfluxDB it can be found in the [cloud platform] itself.

Reviewer: ```suggestion
For CrateDB, the credentials are displayed at time of cluster creation.
For InfluxDB, they can be found in the [cloud platform] itself.
```

Now, same as before, import data from InfluxDB bucket/measurement into 
CrateDB schema/table.
:::{code} shell
ctk load table \
  "influxdb2://9f..06:T2..==@eu-central-1-1.aws.cloud2.influxdata.com/testdrive/demo?ssl=true" \
  --cratedb-sqlalchemy-url="crate://admin:dZ..qB@purple-shaak-ti.eks1.eu-west-1.aws.cratedb.net:4200/testdrive/demo?ssl=true"
:::

::: {note}
Note the **necessary** `ssl=true` query parameter at the end of both database connection URLs
when working on Cloud-to-Cloud transfers.
:::

> And verify that relevant data has been transferred to CrateDB.

Reviewer: ```suggestion
Verify that relevant data has been transferred to CrateDB.
```
